### ISSUE TITLE
Unify census geo data pipeline with what is needed for OEPS

### ISSUE THREAD

**Issue by mradamcox** [healthyregions/oeps#34]
Currently, the census.py operations will download all shapefiles from the US Census FTP for a given year (2010 or 2018), merge them, standardize some fields, export to a variety of formats, `geojson`, `shp`, and `pmtiles`, and then upload these files to S3. However, we are not yet using these in the OEPS download packages, nor in the explorer map. To unify this, the following changes should be made:

- [x] Fix URL lists for FTP (I will do this)
- [x] Change the upload S3 paths to be within the /oeps/ prefix (may be good to make an argument for prefix to allow upload to different directories) (I will do this)
- [ ] Ensure the `geojson` output, which we will be using for State and County layers, has the necessary fields (@bucketteOfIvy )
- [ ] Load these two geojson files for the map directly from S3, not from local geojson, making sure to use the correct year (@bodom0015)

Once this is all pulled together, we can explore other ways of handling the ZCTA and Tract layers without relying on Mapbox.

**Comment by mradamcox** [comment]
Looking further into this, there are a few complicating factors I didn't realize at first.

- The `state.geojson` and `county.geojson` files that are currently used by the explorer web map are heavily generalized, greatly reducing the file size. For example, where the cartographic boundary 500k file for counties that we currently have in the pipeline are ~30mb, the explorer currently uses a 1.5mb file for this.
- The process that loads these geojson files and puts them on the map cannot accept a dropped in http url for the source data, and while I'm sure we could alter it to do so, the question of file size is much more important--regardless of how they are loaded, 30mb is much too large of course.
- Ideally, we could use PMTiles for all geometry layers (we already have these files ready to be used), and could use [Matico-Platform/deck.gl-pmtiles](https://github.com/Matico-Platform/deck.gl-pmtiles). Will require some experimentation though...

**Comment by mradamcox** [comment]
Closing this now, as we won't really get to it in the way that I imagined. What would need to happen is to modify the census data pipeline to use the 20m geometries (separate task that will take a bit of time) and to either 1) modify this code base to either read from PMTiles for the tract and zcta boundaries, or 2) create an upload pipeline directly to Mapbox, which I'm not sure is possible anyway.

For now, I've just recreated the state and county GeoJSON files with census 20m datasets, and recreated the Mapbox tilesets to include HEROP_IDs for the joins.
